packages/website/pages/docs/concepts/architecture-considerations.md

```diff
@@ -42,5 +42,34 @@ Meanwhile, we're continuing to work on a flexible public key based authenticatio
 
 You can track the progress of this work by following [the "Signed uploads w/ UCAN" GitHub issue](https://github.com/nftstorage/nft.storage/issues/851). If you have feedback on the design, please comment on that issue and let us know. 
 
+## Retrieving data for end users
+
+For NFT platforms that need to retrieve NFT data on behalf of others, there are a few things you can do to ensure a quality user experience.
+
+When displaying an NFT that uses IPFS to link to off-chain data, you have several options for retrieval.
+
+### HTTP gateways
+
+IPFS is built around a peer-to-peer content sharing protocol that allows any computer to provide content to anyone that requests it. This provides [many benefits][concepts-decentralized-storage], but not all computing environments can fully support the peer-to-peer networking paradigm.
+
+While some browsers like [Brave](https://brave.com) offer [native IPFS support][brave-ipfs], other browsers are restricted to HTTP and its related protocols, which makes engaging directly with the peer-to-peer protocol more difficult.
+
+As a platform operator, you can support all browsers by using an IPFS gateway to serve NFTs to your users via HTTP. Because NFT.Storage and other services following [best practices for NFT data on IPFS][ipfs-docs-nft-best-practices] use `ipfs://` URLs instead of `https://`, your web application should support re-writing IPFS URLs to target the gateway of your choice.
```

Comment:
Maybe it's worth mentioning that as `ipfs://` gets more widely adopted rewriting would become less necessary and that is why you want to put `ipfs://` instead of `https://`, not sure if it can fit in few enough words to avoid distraction.

Maybe Opera is also worth mentioning